agr_java_core/src/main/java/org/alliancegenome/neo4j/repository/AlleleIndexerRepository.java

```diff
@@ -130,6 +133,14 @@ public AlleleDocumentCache getAlleleDocumentCache(String species) {
         return getMapSetForQuery(query, "a.primaryKey", "phenotype.phenotypeStatement", getSpeciesParams(species));
     }
 
+    public Map<String, Set<String>> getRelatedVariants(String species) {
+        String query = "MATCH (species:Species)--(:Gene)-[:IS_ALLELE_OF]-(a:Allele)--(v:Variant) ";
```

Comment:
Might want to specify the relationship between `(a:Allele)--(v:Variant)` don't know if this query has performance issues or not?

Reply:
Doesn't look like it does, seems to complete in about a second happily.